pyiron/atomistics/structure/pyscal.py

```diff
@@ -5,54 +5,166 @@
 from sklearn import cluster
 from pyiron.lammps.structure import UnfoldingPrism
 from pyiron.atomistics.structure.atoms import pyiron_to_ase
+import numpy as np
 
 try:
     import pyscal.core as pc
 except ImportError:
     pass  # pyscal is currently not available on windows
 
-__author__ = "Jan Janssen"
+__author__ = "Sarath Menon, Jan Janssen"
 __copyright__ = (
     "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
     "Computational Materials Design (CM) Department"
 )
 __version__ = "1.0"
-__maintainer__ = "Jan Janssen"
-__email__ = "janssen@mpie.de"
+__maintainer__ = "Sarath Menon"
+__email__ = "sarath.menon@rub.de"
 __status__ = "development"
 __date__ = "Nov 6, 2019"
 
 
-def get_steinhardt_parameter_job(job, cutoff=3.50, n_clusters=2, q=[4, 6]):
+def get_steinhardt_parameter_job(job, neighbor_method="cutoff", cutoff=0, n_clusters=2, 
+                                 q=[4, 6], averaged=False, clustering=True):
+    """
+    Calculate Steinhardts parameters
+
+    Args:
+        job (job): pyiron job
+        neighbor_method (str) : can be ['cutoff', 'voronoi']
+        cutoff (float) : can be 0 for adaptive cutoff or any other value
+        n_clusters (int) : number of clusters for K means clustering
+        q (list) : can be from 2-12, the required q values to be calculated
+        averaged (bool) : If True, calculates the averaged versions of the parameter
+        clustering (bool) : If True, cluster based on the q values
+
+    Returns:
+        q (list) : calculated q parameters
+    
+    """
     return get_steinhardt_parameter_structure(
-        structure=job.get_stucture(), 
```

Comment:
`job.structure` is the initial structure and `job.get_structure(iteration_step=-1)` returns a structure of the job, only for a static calculation these are the same, for a minimization or an molecular dynamics trajectory they are not. I would therefore recommend to add `iteration_step=-1` as additional argument.

Reply:
> The travis part - I do not know why it fail, it seems to be a timeout issue. The AppVeyor part failed because it is Windows based and pyscal is not available for windows right?

In theory, it should work, but no I have not tested.

Reply:
So `job.structure` and `job.get_structure()` returns the initial configuration? Okay, I will add the additional arguments

Reply:
No, `job.structure` is the initial structure, we basically set it as a property. To get the structure at the end of the calculation or any step of the calculation we have to use `job.get_structure()`

Reply:
The new commit adds iteration step as an argument